src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -11,6 +11,9 @@
 from marqo.core.unstructured_vespa_index.unstructured_document import UnstructuredVespaDocument
 from marqo.core.vespa_index import VespaIndex
 from marqo.exceptions import InternalError, InvalidArgumentError
+from marqo.logging import get_logger
+
+logger = get_logger(__name__)
```

Comment:
Why did you add this? It isn't used

Reply:
Removed! I just added it for future ease of use.